azure-pipelines.yml

```diff
@@ -122,7 +122,7 @@ stages:
       - job: Linux
         pool:
           ${{ if eq(variables['System.TeamProject'], 'public') }}:
-            name: NetCore-Public
+            name: NetCore-Svc-Public
             demands: ImageOverride -equals Build.Ubuntu.2204.Amd64.Open
           ${{ if eq(variables['System.TeamProject'], 'internal') }}:
```

Comment:
I'm a little curious if internal CI should run in -Svc pool for the same purpose or not.

Reply:
It should; from my checking though it already does.